packages/core/src/attestation/Attestation.ts

```diff
@@ -79,6 +87,18 @@ export default class Attestation implements IAttestation {
    * @param claimHash - The hash of the claim that corresponds to the attestation to remove.
    * @param maxDepth - The number of levels to walk up the delegation hierarchy until the delegation node is found.
    * @returns A promise containing the unsigned SubmittableExtrinsic (submittable transaction).
+   * @example ```javascript
+   * Attestation.remove('0xd8024cdc147c4fa9221cd177', 3).then((removalExtrinsic) => {
+   *   // The attestation removal tx was created, and it can now be signed by the attestation owner.
+   *   const attestationOwnerDid = did;
```

Comment:
C: This does not seem to be needed

Reply:
https://github.com/KILTprotocol/sdk-js/pull/434/commits/974293be648c3fafc24709faf09638530614cb5c